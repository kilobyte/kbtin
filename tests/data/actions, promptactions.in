#hook close {#showme FIN;#end}
#act {foo} {#showme <foo>}
#promptact {fo} {#showme <fo...>}
#run a {sh -c "printf 'fo';sleep 2;echo 'o'"}
