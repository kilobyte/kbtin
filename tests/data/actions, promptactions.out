<fo...>
<fo...>
<foo>
foo
FIN
